com.discordapp.DiscordCanary.json

```diff
@@ -20,7 +20,8 @@
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--talk-name=com.canonical.AppMenu.Registrar",
         "--talk-name=com.canonical.indicator.application",
-        "--talk-name=com.canonical.Unity.LauncherEntry"
+        "--talk-name=com.canonical.Unity.LauncherEntry",
+        "--own-name=org.kde.*"
```

Comment:
This...should definitely not be taking ownership access to the *entire namespace*, should it instead be `org.kde.StatusNotifier`?

Worth noting that StatusNotifier are also inherently broken on flatpaks due to encoding the PID in the D-Bus name, so I'm not sure how worthwhile this is anyway...

Reply:
No, that's not the correct permission, electron/chromium use dbus menus now that use a completely different interface

Reply:
Either way, this really should not give access to the entire `org.kde` namespace, which opens up a massive amount of sensitive things from your system. You can remove this permission and run it with `flatpak --log-session-bus` to see what name is actually being used.

Reply:
https://github.com/electron/electron/issues/37053
the name is based on the PID that is impossible unfortunately, the fix has been made upstream but we cannot know when discord will update their electron again, so this is the only fix for now unfortunately :(